### PR TITLE
Clue continue, favalch in alch, black mask in fightcaves, and minor etc

### LINKED ISSUE #2209: Ring of the Gods (i) doesn't work as a normal Ring of the Gods for lizardman shamans.
## 🐛 Bug Report
just the title

### PR DESCRIPTION
**Pull request by Lilylicious**
### Description:

* You can now press Y/continueCharacter to open clue boxes you just got from a clue trip
* Solo CoX are repeatable, both CM and not.
* +alch now selects the highest value +favalch item if no item name is provided
* Black mask now boosts in fight caves even from bank, consistent with +k
* Chilli and Egg potatoes now eatable
* Ring of the Gods (i) now boosts shamans
* Grand exchange message remove notification now mentions if the reason was lack of mandatory words
* Slayer info when killing Jad now matches +k, giving info about slayer task streaks and points. Slayer XP per hour is now also shown.

### Other checks:

-   [X] I have tested all my changes thoroughly.

closes #2209

### CODE REVIEW

File: src/tasks/minions/minigames/raidsActivity.ts [hunk 1]
```diff
@@ -108,5 +110,30 @@ export default class extends Task {
...
+
+					allUsers[0].log('continued trip of solo CoX');
+					return this.client.commands.get('raid')!.run(res, ['solo']);
+				},
+				undefined,
+				data,
+				null
+			);
+		}
 	}
```
Comment (gc) [inline src/tasks/minions/minigames/raidsActivity.ts:138]: Isn't this going to cause 2 messages to be sent? you should do only sendtochannelid in masses, only handetripfinish in solos.

Also, please don't make PRs like this with lots of unrelated changes. Making 10 different PRs with small changes is a lot easier.

**Review by gc — CHANGES_REQUESTED** [review]

File: src/tasks/minions/minigames/raidsActivity.ts (hunk 1 above)
Comment (Lilylicious) [reply src/tasks/minions/minigames/raidsActivity.ts]: Right you are on both counts! Future PRs won't have the same issue.

**Comment by Lilylicious** [comment]
Closing this. Will create new PRs for individual features.
